src/tyr/actor.cc

```diff
@@ -77,6 +77,10 @@ actor_t::route(const std::string& request_str, const std::function<void()>* inte
   return bytes;
 }
 
+std::string actor_t::route(const std::string& request_str) {
+  return actor_t::route(request_str, NULL, NULL);
+}
+
```

Comment:
I realize this pattern is ugly and repetitive.. hints or a little code sample would be appreciated